src/algorithms/tracking/AmbiguitySolverConfig.h

```diff
@@ -0,0 +1,17 @@
+// Created by Minjung Kim (minjung.kim@lbl.gov)
+// Subject to the terms in the LICENSE file found in the top-level directory.
```

Comment:
```suggestion
// SPDX-License-Identifier: LGPL-3.0-or-later
// Copyright (C) 2024 Minjung Kim
```

Reply:
@minjungkim12 Can you please not mark things as resolved when comments are not resolved? Somehow this is still/again the wrong header.